Client/Frontend/Browser/URLBarView.swift

```diff
@@ -299,25 +300,25 @@ class URLBarView: UIView {
     }
 
     func updateTabCount(count: Int) {
-
         // make a 'clone' of the tabs button
         let newTabsButton = InsetButton()
+        newTabsButton.setTranslatesAutoresizingMaskIntoConstraints(false)
         newTabsButton.setTitleColor(AppConstants.AppBackgroundColor, forState: UIControlState.Normal)
         newTabsButton.titleLabel?.layer.backgroundColor = UIColor.whiteColor().CGColor
         newTabsButton.titleLabel?.layer.cornerRadius = 2
         newTabsButton.titleLabel?.font = AppConstants.DefaultSmallFontBold
         newTabsButton.titleLabel?.textAlignment = NSTextAlignment.Center
+        newTabsButton.setTitle(count.description, forState: .Normal)
+        addSubview(newTabsButton)
         newTabsButton.titleLabel?.snp_makeConstraints { make in
             make.size.equalTo(URLBarViewUX.TabsButtonHeight)
         }
-
-        if let anchor = tabsButton.titleLabel?.layer.anchorPoint {
-            newTabsButton.titleLabel?.layer.anchorPoint = anchor
+        newTabsButton.snp_makeConstraints { make in
+            make.centerY.equalTo(self.locationContainer)
+            make.trailing.equalTo(self)
+            make.width.height.equalTo(AppConstants.ToolbarHeight)
```

Comment:
You could use `make.size` instead of `make.width.height` if you want to save a few characters :)